x-pack/plugin/ilm/qa/multi-node/build.gradle

```diff
@@ -1,19 +1,26 @@
+import org.elasticsearch.gradle.test.RestIntegTestTask
+
+apply plugin: 'elasticsearch.testclusters'
 apply plugin: 'elasticsearch.standalone-rest-test'
 apply plugin: 'elasticsearch.rest-test'
 
 dependencies {
     testCompile project(path: xpackProject('plugin').path, configuration: 'testArtifacts')
 }
 
-integTestRunner {
+File repoDir = file("$buildDir/testclusters/repo")
+
+integTest.runner {
     /* To support taking index snapshots, we have to set path.repo setting */
-    systemProperty 'tests.path.repo', new File(buildDir, "cluster/shared/repo")
+    systemProperty 'tests.path.repo', repoDir
 }
 
-integTestCluster {
-    numNodes = 4
-    clusterName = 'ilm'
+testClusters.integTest {
+    distribution = 'DEFAULT'
+    numberOfNodes = 4
+    //clusterName = 'ilm'
```

Comment:
Is there a reason we can't name the cluster `ilm`? If so we should just remove this line